recipe/meta.yaml

```
@@ -1,45 +1,59 @@
 {% set name = "cuda-python" %}
-{% set version = "11.8.1" %}
+{% set version = "12.0.0" %}
 {% set major_version = version.split(".")[0] %}
 {% set minor_version = version.split(".")[1] %}
 {% set patch_version = version.split(".")[2] %}
+{% set target_name = "x86_64-linux" %}  # [linux64]
+{% set target_name = "ppc64le-linux" %}  # [ppc64le]
+{% set target_name = "sbsa-linux" %}  # [aarch64]
+{% set target_name = "x64" %}  # [win]
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/NVIDIA/cuda-python/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: ef54ada5d607387f59950cfb3c3852e5816b62824402d289227b8066e8353489
+  sha256: 7df0f84c4b6210112353374a7d992bfd5bba9c97358b39ea03b61bd634ce9c8a
 
 build:
-  number: 2
-  skip: true  # [py<37 or osx or cuda_compiler_version != "11.8"]
+  number: 0
+  skip: true  # [aarch64 or ppc64le or win or osx or cuda_compiler_version != "12.0"]
```

aarch64 and ppc64le and windows are getting skipped?

---

```suggestion
  # TODO: aarch64, ppc64le, and Windows are not yet supported by CUDA 12 on conda-forge.
  # These platforms can be added in a later build.
  skip: true  # [aarch64 or ppc64le or win or osx or cuda_compiler_version != "12.0"]
```

---

We could also file issues on the feedstock about these needs. Maybe one for `aarch64` & `ppc64le` (as they are closely related) and one for Windows? These could then be linked in this comment

---

Yes please, as long as there's a way to track back to progress on the various platforms.

---

Filed issues for adding all of these on CUDA 12. Think we can include them like so in the comment

```suggestion
  # TODO: Add `linux_aarch64`, `linux_ppc64le`, and `win_64` once supported on CUDA 12 inn conda-forge.
  # xref: https://github.com/conda-forge/cuda-python-feedstock/issues/36
  # xref: https://github.com/conda-forge/cuda-python-feedstock/issues/37
  # xref: https://github.com/conda-forge/cuda-python-feedstock/issues/38
  skip: true  # [aarch64 or ppc64le or win or osx or cuda_compiler_version != "12.0"]
```

xref: https://github.com/conda-forge/cuda-python-feedstock/issues/36
xref: https://github.com/conda-forge/cuda-python-feedstock/issues/37
xref: https://github.com/conda-forge/cuda-python-feedstock/issues/38

---

That works for me, resolving